dotcom-rendering/README.md

```diff
@@ -188,13 +150,3 @@ We recommend you update your workspace settings to automatically fix formatting
     	"source.fixAll.eslint": true
     }
     ```
-
-If you prefer not to use an editor like VSCode then you can use the following commands to manage formatting and (try to fix) linting errors:
-
--   `make fix` &rarr; Checks and fixes prettier and linting issues
-
-## Thanks
-
-<a href="https://www.chromaticqa.com/"><img src="https://cdn-images-1.medium.com/letterbox/147/36/50/50/1*oHHjTjInDOBxIuYHDY2gFA.png?source=logoAvatar-d7276495b101---37816ec27d7a" width="120"/></a>
-
-Thanks to [Chromatic](https://www.chromaticqa.com/) for providing the visual testing platform that helps us catch unexpected changes on time
```

Comment:
Chromatic used to offer us a free service if we called them out in our README. Since we now pay Chromatic handsomely, we no longer need to do this.